src/declaration.d

```diff
@@ -1016,6 +1016,8 @@ extern (C++) class VarDeclaration : Declaration
 {
     Initializer _init;
     uint offset;
+    uint sequenceNumber;            // order the variables are declared
```

Comment:
order the variables are declared???
Thought you were the native speaker :).
Did you mean order the declaration of variables?